src/careamics/models/lvae/layers.py

```diff
@@ -51,13 +50,13 @@ def __init__(
         self,
         channels: int,
         nonlin: Callable,
-        kernel: Union[int, Iterable[int]] = None,
+        conv_strides: tuple[int] = (2, 2),
```

Comment:
replace with `conv_dims` once defined :)

Reply:
But it's not a stride it's a kernel size if I'm not mistaken